src/stdlib_ascii.f90

```diff
@@ -145,13 +145,15 @@ pure logical function is_printable(c)
         character(len=1), intent(in) :: c !! The character to test.
         integer :: ic
         ic = iachar(c)                    ! '~'
-        is_printable = c >= ' ' .and. ic <= int(z'7E')
+        is_printable = ic >= iachar(' ') .and. ic <= int(z'7E')
     end function
 
     !> Checks whether `c` is a lowercase ASCII letter (a .. z).
     pure logical function is_lower(c)
         character(len=1), intent(in) :: c !! The character to test.
-        is_lower = (c >= 'a') .and. (c <= 'z')
+        integer :: ic
+        ic = iachar(c)
+        is_lower = (ic >= iachar('a')) .and. (ic <= iachar('z'))
```

Comment:
Is this parenthesis really needed here?
```suggestion
        is_lower = ic >= iachar('a') .and. ic <= iachar('z')
```

Reply:
I noticed the entire code has those extra parenthesis except the change above so... I think we should either remove all of them or add some extra parenthesis in the function `is_printable` above so it matches the style of the other ones.